### PR TITLE
[IN-2469] xcbeautify download from GitHub

### PR DESCRIPTION
**Pull request by peterheja-bitrise**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [-] I updated the corresponding tests if there were any.
- [-] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [x] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [x] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

File: roles/weekly-update-shared/tasks/tools-from-artifacts.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    checksum: sha256:75e9d345a68a759ab268f0ecef4373a0ae590b07939096a076bdb25f20e69143
+    dest: /tmp/xcbeautify.zip
+  when: xcbeautify_version.rc != 0 or xcbeautify_version.stdout != "0.8.1"
+
+- name: unarchive XCBeautify binary
+  unarchive:
+    src: /tmp/xcbeautify.zip
+    dest: /usr/local/bin/
+    owner: "{{ param_user }}"
+    mode: "0644"
```
Comment (bitrise-tom) [inline roles/weekly-update-shared/tasks/tools-from-artifacts.yml:20]: this isn't very executable. Should it be?

File: roles/weekly-update-shared/tasks/tools-from-artifacts.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    remote_src: true
+  become: true
+  when: xcbeautify_version.rc != 0 or xcbeautify_version.stdout != "0.2.21"
```
Comment (bitrise-tom) [inline roles/weekly-update-shared/tasks/tools-from-artifacts.yml:23]: this looks like you copied something wrongly ;)

**Review by bitrise-tom — CHANGES_REQUESTED** [review]

**Review by bitrise-tom — DISMISSED** [review]
looks better :)

**Review by fehersanyi-bitrise — APPROVED** [review]
I like where this is going!

**Review by bitrise-tom — APPROVED** [review]

**Review by fehersanyi-bitrise — APPROVED** [review]

**Pull request merged**